docker-jans-shibboleth/scripts/shib_setup.py

```
@@ -0,0 +1,235 @@
+        
+        jetty_base = os.environ.get("JETTY_BASE", f"{SHIBBOLETH_HOME}/jetty")
+        
+        start_ini = f"""
+--module=server
+--module=http
+--module=deploy
+--module=webapp
+--module=resources
+jetty.http.port=8080
+jetty.deploy.scanInterval=0
+"""
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)